pkg/bitcoin/chain.go

```diff
@@ -20,12 +20,12 @@ type Chain interface {
 	// rejected eventually.
 	BroadcastTransaction(transaction *Transaction) error
 
-	// GetCurrentBlockNumber gets the number of the current block. If the
-	// current block was not determined, this function returns an error.
-	GetCurrentBlockNumber() (uint, error)
+	// GetLatestBlockHeight gets the height of the latest block (tip). If the
+	// latest block was not determined, this function returns an error.
+	GetLatestBlockHeight() (uint, error)
```

Comment:
@lukasz-zimnoch I hope you're fine with this rename.

The function expects the "chain tip", so the most recent mined block. I propose we use "latest", as "current" may be confusing and suggest that the block is being currently mined.

Also, I replaced "block number" with "block height", as it seems to be widely used in the Bitcoin ecosystem.